.eslintrc

```diff
@@ -13,6 +13,9 @@
 			"extends": "@martin-kolarik/eslint-config/typescript-type-checking",
 			"parserOptions": {
 				"project": true
+			},
+			"rules": {
+				"@typescript-eslint/consistent-type-imports": "error"
```

Comment:
This rule isn't enabled by default because adding the type annotation has no value at all and just adds noise to the import declarations. If we want to add any rule about this, it should be banning the `type` imports instead.